samples/synchronization/README.rst

```diff
@@ -12,6 +12,11 @@ and use sleep requests and semaphores to control the rate at which messages
 are generated. This demonstrates that kernel scheduling, communication,
 and timing are operating correctly.
 
+And this application can also be used to demonstrate the tracing feature. The
```

Comment:
sorry if I confused you, I did not mean for you to reuse the existing sample, I was saying that you could reuse the code. I prefer to leave this sample as simple as possible, anything with tracing can be added in a tracing specific sample that can be extended to show tracing specific features.

You could use this one if you wish: `tests/subsys/debug/tracing`